src/Symfony/Component/Validator/Constraints/Negative.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ */
+class Negative extends LessThan
+{
+    public $message = 'This value should be negative.';
+
+    public function __construct(array $options = null)
```

Comment:
Reason for most constraints not requiring array is because in such case they treat it as default option. However, these new constraints can't really have default option.